README.md

```diff
@@ -15,7 +15,9 @@ Instructions to install on [bosh-lite](https://github.com/cloudfoundry/bosh-lite
 bosh target BOSH_DIRECTOR_HOST
 git clone https://github.com/cloudfoundry-incubator/app-autoscaler-release
 cd app-autoscaler-release
+source .envrc
```

Comment:
The main purpose is to se the GOPATH so that we can use gosub. 
I will put the GOPAHT setting in script